target/linux/ath79/image/generic.mk

```diff
@@ -330,6 +330,17 @@ define Device/comfast_cf-wr650ac-v2
 endef
 TARGET_DEVICES += comfast_cf-wr650ac-v2
 
+define Device/comfast_cf-wr752ac-v1
+  SOC := qca9531
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-WR752AC
+  DEVICE_VARIANT := v1
+DEVICE_PACKAGES := kmod-leds-gpio kmod-usb2 kmod-ath10k-ct ath10k-firmware-qca9888-ct \
+	-uboot-envtools swconfig
```

Comment:
Still haven't addressed my swconfig-related comment.

Reply:
swconfig is not there by default, I had to add it. It is also used in cf-fw when in repeater mode (eth port is on phy4 of the switch, by hardware separated from the rest). But since there are other preset modi on firmware (as AP, router), it is up to the user to configure it. I can't replicate whole Comfast scripts.

Reply:
There it is: https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/Makefile#L16

Reply:
OK, I must have confounded with another platform. As said, I made that a few months ago. I also did  remove the kmod-leds-gpio, LEDs do work without too. But as far as I remember, taking out USB disables the PCIe (your comment from yesterday).